packages/cli/src/commands/login/index.ts

```diff
@@ -67,20 +68,34 @@ export default async function login(client: Client): Promise<number> {
     return result;
   }
 
+  const isNewLogin = !client.authConfig.token;
+
+  // Save the user's authentication token to the configuration file.
+  client.authConfig.token = result.token;
+
   // If the token was upgraded (not a new login), then don't modify
   // the current scope.
-  if (!client.authConfig.token) {
+  if (isNewLogin) {
     if (result.teamId) {
       // SSO login, so set the current scope to the appropriate Team
       client.config.currentTeam = result.teamId;
     } else {
-      delete client.config.currentTeam;
+      let user = null;
+      try {
+        user = await getUser(client);
+      } catch (err: unknown) {
+        // Shouldn't happen since we just logged in
+        output.error('Not able to load user');
+        return 2;
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** The exit code `2` should only be used for help output. Let's change this to `1`.

`‏
```suggestion
        return 1;
```